Grinnell-DB-iOS/ResultsViewController.m

```diff
@@ -19,282 +19,281 @@ @implementation ResultsViewController
-            index++;
-        }
-        NSString *compositeTitle = [titleArray objectAtIndex:0];
-        
-        for (int i = 1; i < titleArray.count; i++) {
-            compositeTitle = [compositeTitle stringByAppendingString:[NSString stringWithFormat:@"/%@", [titleArray objectAtIndex:i]]];
-        }
-        
-        compositeTitle = [compositeTitle stringByTrimmingCharactersInSet:[NSCharacterSet whitespaceAndNewlineCharacterSet]];
-        majorLbl.text = compositeTitle;
-        classLbl.text = dept;
-        statusLbl.text = @"Fac/Staff";
-    } else {
-        NSUInteger index = [tempPerson.attributes indexOfObject:@"Title"];
-        if (NSNotFound != index) {
-            NSString *title = [tempPerson.attributeVals objectAtIndex:index];
-            majorLbl.text = title;
-            classLbl.text = [tempPerson.attributeVals objectAtIndex:[tempPerson.attributes indexOfObject:@"Department"]];
-        } else {
-            majorLbl.text = @"Data unavailable off campus";
-            classLbl.text = @"Unavailable";
-        }
-        statusLbl.text = @"Unavailable";
-    }
-    
-    nameLbl.text = [NSString stringWithFormat:@"%@, %@", last, first];
-    usernameLbl.text = username;
-    
-    return cell;
+    statusLbl.text = @"Unavailable";
+  }
+
+  nameLbl.text = [NSString stringWithFormat:@"%@, %@", last, first];
+  usernameLbl.text = username;
+
+  return cell;
 }
 
 #pragma mark - Table view delegate
 - (void)tableView:(UITableView *)tableView didSelectRowAtIndexPath:(NSIndexPath *)indexPath {
-    [self performSegueWithIdentifier:@"PushToProfile" sender:self];
+  [self performSegueWithIdentifier:@"PushToProfile" sender:self];
 }
 
 #pragma mark UIAlertViewDelegate Methods
 // Called when an alert button is tapped.
 - (void)alertView:(UIAlertView *)alertView clickedButtonAtIndex:(NSInteger)buttonIndex {
-    return;
+  return;
 }
 
 - (void)showNoNetworkAlert {
-    UIAlertView *network = [[UIAlertView alloc]
-                            initWithTitle:@"No Network Connection"
-                            message:@"Turn on cellular data or use Wi-Fi to access the server"
-                            delegate:self
-                            cancelButtonTitle:@"OK"
-                            otherButtonTitles:nil
-                            ];
-    [network show];
-}
-
-- (void)showErrorAlert {
-    UIAlertView *error = [[UIAlertView alloc]
-                          initWithTitle:@"An Error Occurred"
-                          message:@"Please try again later"
+  UIAlertView *network = [[UIAlertView alloc]
+                          initWithTitle:@"No Network Connection"
+                          message:@"Turn on cellular data or use Wi-Fi to access the server"
                           delegate:self
                           cancelButtonTitle:@"OK"
                           otherButtonTitles:nil
                           ];
-    [error show];
+  [network show];
+}
+
+- (void)showErrorAlert {
```

Comment:
Would it be better for this to be showGeneralErrorAlert or showUnknownErrorAlert?